js/binance.js

```diff
@@ -867,10 +866,12 @@ module.exports = class binance extends Exchange {
             'precisionMode': DECIMAL_PLACES,
             // exchange-specific options
             'options': {
+                'fetchMarkets': [ 'spot', 'linear', 'inverse' ],
```

Comment:
I think , dict would be better than array for setting values (we rarely use array)
```suggestion
                'fetchMarkets': {
                    'spot': true,
                    'linear': true,
                    'inverse': true,
                },
```

that way, user can come back and reset `true/false` for any of them, as opposed to array, where user has just to totally erase the slug from the list, and then might not remember what to place back.